omim2obo/main.py

```diff
@@ -164,63 +164,69 @@ def omim2obo(use_cache: bool = False):
         if omim_type == OmimType.HERITABLE_PHENOTYPIC_MARKER:  # %
             graph.add((omim_uri, BIOLINK['category'], BIOLINK['Disease']))
-        elif omim_type == OmimType.GENE or omim_type == OmimType.HAS_AFFECTED_FEATURE:  # * or +
-            use_abbrev_over_label = True
+        elif is_gene:  # * or +
             graph.add((omim_uri, RDFS.subClassOf, SO['0000704']))  # gene
             graph.add((omim_uri, MONDO.exclusionReason, MONDO.nonDisease))
             graph.add((omim_uri, BIOLINK['category'], BIOLINK['Gene']))
         elif omim_type == OmimType.PHENOTYPE:
             graph.add((omim_uri, BIOLINK['category'], BIOLINK['Disease']))  # phenotype ~= disease
         elif omim_type == OmimType.SUSPECTED:
             graph.add((omim_uri, MONDO.exclusionReason, MONDO.excludeTrait))
-        else:
-            pass
 
-        if use_abbrev_over_label and abbrev:
-            graph.add((omim_uri, RDFS.label, Literal(abbrev)))
+        # Alternative rdfs:label for genes
+        if is_gene and pref_symbols:
+            gene_label_err = 'Warning: Only 1 symbol picked for label for gene term, but there were 2 to choose' \
+                 f'from. Unsure which is best. Picking the first.\nhttps://omim.org/entry/{omim_id} - {pref_symbols}'
+            if len(pref_symbols) > 1:
+                LOG.warning(gene_label_err)  # todo: decide the best way to handle these situations
+            graph.add((omim_uri, RDFS.label, Literal(pref_symbols[0])))
         else:
-            graph.add((omim_uri, RDFS.label, Literal(label_cleaner.clean(label))))
-
-        # Reify on abbreviations. See: https://github.com/monarch-initiative/omim/issues/2
-        exact_labels = [s.strip() for s in label.split(';')]
-        if len(exact_labels) > 1:
-            abbr = exact_labels.pop()
-            graph.add((omim_uri, oboInOwl.hasExactSynonym, Literal(abbr)))
-            # Acronym: uppercase, no numbers, no whitespace, <10 chars
-            if abbr.isupper() and not any(char.isspace() for char in abbr) and len(abbr) < 10:
-                axiom = BNode()
-                graph.add((axiom, RDF.type, OWL.Axiom))
-                graph.add((axiom, OWL.annotatedSource, omim_uri))
-                graph.add((axiom, OWL.annotatedProperty, oboInOwl.hasExactSynonym))
-                graph.add((axiom, OWL.annotatedTarget, Literal(abbr)))
-                graph.add((axiom, oboInOwl.hasExactSynonym, MONDONS.abbreviation))
-        for exact_label in exact_labels:
-            graph.add((omim_uri, oboInOwl.hasExactSynonym, Literal(label_cleaner.clean(exact_label, abbrev))))
-        for label in other_labels:
-            graph.add((omim_uri, oboInOwl.hasExactSynonym, Literal(label_cleaner.clean(label, abbrev))))
+            graph.add((omim_uri, RDFS.label, Literal(label_cleaner.clean(pref_title))))
+
+        # todo: .clean() should accept all symbols/abbrevs (preferred, alt, included), else [] instead of str/None
```

Comment:
I don't think `.clean()` ever needs to be used on symbols/abbrevs since no case changes need to be made to these values.

Is this a reference to the possible future work about symbols/abbrevs being used to inform the casing of symbols found in preferred Titles as you pointed out in our discussion? If yes, then can you make the comment more clear to indicate that?

Reply:
Ok, I think that comment is for issue 129. If it said something like `.clean()` should accept a list of abbreviations vs. only one abbreviation value then that would be more clear.

Reply:
Yes, you're correct. But it is actually not just accepting a list of abbreviations. It is that it should pass all abbreviations of any of preferred, alt, or included. Currently, it is only passing the 1st preferred abbreviation.

I updated the comment to:
> todo: .clean()/.cleanup_label() 2nd param `explicit_abbrev` should be List[str] instead of str. And below, should pass all symbols/abbrevs from each of preferred, alt, included each time it is called. If no symbols for given term, should pass empty list. See: https://github.com/monarch-initiative/omim/issues/129